torch/csrc/jit/runtime/symbolic_shape_registry.cpp

```diff
@@ -66,6 +66,9 @@ static const OperatorMap<std::string>& get_schema_to_function_graph() {
       {"aten::rsub.Tensor(Tensor self, Scalar other, Scalar alpha=1) -> Tensor", "unary"},
       {"aten::dropout(Tensor input, float p, bool train) -> Tensor", "unary"},
       {"aten::adaptive_avg_pool2d(Tensor self, int[2] output_size) -> Tensor", "adaptive_avg_pool2d"},
+      {"aten::gelu(Tensor self) -> Tensor", "unary"},
```

Comment:
These three ops are now in shape analysis via https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/runtime/symbolic_shape_registry_util.cpp#L39 so we don't need them here